website/docs/howto.md

```diff
@@ -75,10 +75,10 @@ kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/
 Note the `match` field, which defines the scope of objects to which a given constraint will be applied. It supports the following matchers:
 
    * `kinds` accepts a list of objects with `apiGroups` and `kinds` fields that list the groups/kinds of objects to which the constraint will apply. If multiple groups/kinds objects are specified, only one match is needed for the resource to be in scope.
-   * `scope` accepts `*`, `Cluster`, or `Namespaced` which determines if cluster-scoped and/or namespace-scoped resources are selected. (defaults to `*`)
+   * `scope` determines if cluster-scoped and/or namespaced-scoped resources are matched.  Accepts `*`, `Cluster`, or `Namespaced`. (defaults to `*`)
```

Comment:
if these changes apply to previous versions (v3.6 and v3.7), can you updates those in `versioned_docs` folder too?

Reply:
Done!